.github/workflows/esp32_build.yml

```diff
@@ -151,26 +151,40 @@ jobs:
       matrix:
         config: [
             esp32buzz,
+            esp32s3empty,
         ]
         gcc: [10]
 
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: 'recursive'
+
+      - name: Register gcc problem matcher
+        run: echo "::add-matcher::.github/problem-matchers/gcc.json"
+
+      - name: Register python problem matcher
+        run: echo "::add-matcher::.github/problem-matchers/python.json"
+
+      - name: Register autotest warn matcher
+        run: echo "::add-matcher::.github/problem-matchers/autotestwarn.json"
+
+      - name: Register autotest fail matcher
+        run: echo "::add-matcher::.github/problem-matchers/autotestfail.json"
+
       - name: Install Prerequisites
         shell: bash
         run: |
           sudo apt-get install git wget libncurses-dev flex bison gperf python3 python3-pip python3-venv python3-setuptools python3-serial python3-gevent python3-cryptography python3-future python3-pyparsing python3-pyelftools cmake ninja-build ccache libffi-dev libssl-dev dfu-util libusb-1.0-0
-          sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+          sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10 2>/dev/null >/dev/null
```

Comment:
Why?  That output might be useful when it fails?!

Reply:
these things never fail, do they. :-)

Reply:
.. also its pretty safe because that's where we install 3.10, and we immediately install 3.11 below as the step that actually matters.